docs/partner_editable/additional_info.adoc

```diff
@@ -74,7 +74,9 @@ TBLPROPERTIES (
   'skip.header.line.count'='1');
 ```
 
-If you have other weather dataset you want to use, just make sure it contains at least below 4 columns.
+If you have another weather dataset you want to use, make sure it contains at least the four columns shown in [___].
+
+//TODO Shivansh, how do we fill in the blank in the previous sentence so that it automatically says "Table 1"? Do we need something like what we use for figures — ':xrefstyle: short' — to make sure we generate the number, not the caption text? (Please let me know so that I can fill in table references on my own in the future. Thanks.)
```

Comment:
If I remember correctly, tables can be x-referenced by using regular anchor syntax to the title of the table, as ascii docs by default creates id for titles, which can be used for cross-referencing, with syntax <<>>

Reply:
For a cross-ref like this to work, we need to create an anchor name. I'll submit a separate PR with this edit. Meanwhile, I've documented this on our [team wiki's Docs 2.0 page](https://w.amazon.com/bin/view/AWS_Quick_Starts/docs2_0/), including this illustration:
![table-cross-ref-asciidoc](https://user-images.githubusercontent.com/61847119/95402821-7fc30980-08c5-11eb-923d-5d9a76c0fa52.png)